phpunit.xml

```diff
@@ -15,10 +15,17 @@
     </testsuites>
 
     <logging>
-        <log type="coverage-html" target="tests/coverage/"/>
+        <log type="coverage-clover" target="build/logs/clover.xml"/>
+        <log type="coverage-php" target="build/logs/coverage.cov"/>
+        <log type="coverage-text" target="build/logs/coverage.txt" showUncoveredFiles="false"/>
+        <log type="coverage-html" target="test/reports/coverage/" showUncoveredFiles="true"/>
```

Comment:
Just how I have always done this, not sure what the effects of changing are, I didn't get mine from a blog, a fellow dev showed me their setup and it worked, unlike reading some articles, so I stuck with it. Not ideal, but the config setup is one of the least important parts of this as it does not affect the test outcome, or the repository